src/layout/elkLayout.ts

```diff
@@ -61,7 +65,7 @@ function mapNode(nodes: NodeData[], edges: EdgeData[], node: NodeData) {
       }))
       : [],
     layoutOptions: {
-      'elk.padding': '[left=50, top=50, right=50, bottom=50]',
+      'elk.padding': `[left=${paddingLeft}, top=${paddingTop}, right=${paddingRight}, bottom=${paddingBottom}]`,
```

Comment:
AHHHH - I forgot about this! How about we make it CSS syntax .... 

`20` or `20 30` or `20 15 10 40`

I did something like this in reaviz here : https://github.com/reaviz/reaviz/blob/master/src/common/utils/dimensions.ts#L30

Reply:
Yes, that sounds good! I will do that. 
Btw, what do you think of the naming? I think a variable name as "innerPadding" or "nestedPadding" could be more self-explaining.

Reply:
Maybe keep it simple - `nodePadding`?